tests/functional/u/used/used_before_assignment_typing.py

```diff
@@ -8,7 +8,62 @@
     if True:  # pylint: disable=using-constant-test
         import math
     import datetime
+    import calendar
+    import zoneinfo
+    import array
+    import types
+    import enum
+    import collections
+    import copy
+    import pprint
+    import heapq
+    import bisect
+    import weakref
+    import numbers
+    import email
+    import json
+    import mailbox
+    import mimetypes
+    import base64
+    import binascii
     from urllib.request import urlopen
+elif input():
+    import calendar, bisect  # pylint: disable=multiple-imports
+    if input() + 1:
+        import heapq
+    elif (enum:=None):
```

Comment:
I would put spaces around the `:=` here and below.

Reply:
Done!